packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -833,19 +833,19 @@ Expected response body:
 }
 ```
 
-### Azure_ResourceManager_Models_CommonTypes_ManagedIdentity_ManagedIdentityTrackedResources_get
+### Azure_ResourceManager_CommonTypes_ManagedIdentity_Property_get
```

Comment:
would we just be able to do `Azure_ResourceManager_CommonTypes_ManagedIdentity_get` and remove `Property` for all of the `ManagedIdentity` sections?

Reply:
Thanks, seems the answer is yes. I found a way to make the namespace `Azure.ResourceManager.CommonTypes` work. Seems I just need to change the `Versions` name.

Downside is the confusion of whether `ManagedIdentity` is resource or a property. 
From dashboard perspective, since we already have the separation of `Resources` and `CommonTypes`, it should be good.
From codegen side, we'll have generated code be like `ManagedIdentity.create()`... Though seems only to be implementation details.